testsuite/features/step_definitions/command_steps.rb

```diff
@@ -894,11 +884,11 @@
 
 When(/^I install package(?:s)? "([^"]*)" on this "([^"]*)"((?: without error control)?)$/) do |package, host, error_control|
   node = get_target(host)
-  if host.include? 'ceos'
+  if rh_host?(host)
```

Comment:
Same question here, where do you set this variable?

Reply:
it's not a variable, and it's in `commonlib.rb`